ocs_ci/ocs/bucket_utils.py

```diff
@@ -139,6 +140,27 @@ def retrieve_test_objects_to_pod(podobj, target_dir):
     return downloaded_objects
 
 
+def check_objects_in_bucket(
+    bucket_name, objects_list, mcg_obj, s3pod, tries=5, delay=10
+):
+    """
+    Checks object list present in bucket and compare it with uploaded object Lists
+    """
+    for i in range(tries):
```

Comment:
Please see if you can use retry method instead - https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py#L8